workers/workers-v2/import-stock-order/import-stock-order-vend.js

```diff
@@ -400,7 +403,7 @@ function mapSpreadSheetDataToOrders(db, orderConfigModel, spreadSheetRows, userM
                         eachSpreadSheetRow.supplierModelId = productFound.supplierModelId;
                         eachSpreadSheetRow.categoryModelId = productFound.categoryModelId;
                         eachSpreadSheetRow.categoryModelName = productFound.categoryModel && productFound.categoryModel.length ? productFound.categoryModel[0].name : 'No Category';
-                        eachSpreadSheetRow.supplyPrice = productFound.supply_price;
+                        eachSpreadSheetRow.supplyPrice = eachSpreadSheetRow.supply_price;
```

Comment:
i think this line is useless now ?

Reply:
Yes ! All the mentioned changes are completed.